### PR TITLE
refactor(validation): move schemas definition to subproperty

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/5?utm_source=stack-comment).


Job Reference: FjKEPjoBOKPqy1Ki6Vcu

**Pull request merged**
